### PR TITLE
all: use Go 1.21 slices, maps instead of x/exp/{slices,maps}

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #8419

### CODE REVIEW

File: ipn/ipnlocal/local.go [hunk 1]
```diff
@@ -27,9 +27,10 @@ import (
 	"sync/atomic"
 	"time"
 
+	"slices"
```
Comment (creachadair) [inline ipn/ipnlocal/local.go:30]: Group with the rest of the stdlib? (Usually fmt is better about this)

File: ipn/serve.go [hunk 2]
```diff
@@ -12,7 +12,8 @@ import (
 	"strconv"
 	"strings"
 
-	"golang.org/x/exp/slices"
+	"slices"
```
Comment (creachadair) [inline ipn/serve.go:15]: Here too.

**Review by creachadair — APPROVED** [review]
There's a handful of places where fmt missed grouping but otherwise LGTM.

File: ipn/ipnlocal/local.go (hunk 1 above)
Comment (bradfitz) [reply ipn/ipnlocal/local.go:30]: Whoops. I thought I'd gotten them all already.

File: portlist/poller.go [hunk 3]
```diff
@@ -13,7 +13,8 @@ import (
 	"sync"
 	"time"
```
Comment (valscale) [inline portlist/poller.go:15]: Rogue newline :)

File: ssh/tailssh/incubator.go [hunk 4]
```diff
@@ -27,11 +27,12 @@ import (
 	"strings"
 	"syscall"
```
Comment (valscale) [inline ssh/tailssh/incubator.go:29]: I think this is another rogue newline, c'mon fmt.

File: types/views/views.go [hunk 5]
```diff
@@ -10,8 +10,9 @@ import (
 	"errors"
 	"net/netip"
```
Comment (valscale) [inline types/views/views.go:12]: Rogue newline

**Review by valscale — APPROVED** [review]
I'm assuming that the flaky test that failed on windows was a flake. Thanks for this!

**Pull request merged**